apps/web/utils/actions/user.ts

```
@@ -10,66 +10,77 @@ import { deleteUserStats } from "@/utils/redis/stats";
-export async function saveOnboardingAnswersAction(onboardingAnswers: {
-  surveyId?: string;
-  questions: any;
-  answers: Record<string, string>;
-}): Promise<ServerActionResponse> {
-  const session = await auth();
-  if (!session?.user.id) return { error: "Not logged in" };
+export const saveOnboardingAnswersAction = withActionInstrumentation(
+  "saveOnboardingAnswers",
+  async (onboardingAnswers: {
+    surveyId?: string;
+    questions: any;
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)